src/app/(auth)/reset-password/page.tsx

```diff
@@ -3,15 +3,17 @@
 import { useState, useEffect } from 'react';
 import { useRouter } from 'next/navigation';
 import TextInput from '@/components/TextInput/index';
-import { H1, H4 } from '@/styles/text';
-import { QuestionsDiv } from '@/app/(auth)/styles';
+import { H1, H4, P } from '@/styles/text';
+import COLORS from '@/styles/colors';
+import { SpacerDiv } from '@/app/(auth)/styles';
 import BigButton from '@/components/BigButton';
 import supabase from '@/api/supabase/createClient';
 
 export default function ResetPassword() {
   const [newPassword, setNewPassword] = useState('');
   const [newPassword2, setNewPassword2] = useState('');
   const [canReset, setCanReset] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
   const { push } = useRouter();
   useEffect(() => {
```

Comment:
nit: can we add a line before `useEffect` lol

Reply:
done!